### PR TITLE
Added actions

### PR DESCRIPTION
**Pull request by stanleySimeon**
# In this milestone I added:

  - Action to add a new book
  - Action to remove a book
  - Action for the categories reducer

### CODE REVIEW

File: src/redux/books/books.js [hunk 1]
```diff
@@ -1,42 +1,45 @@
...
+    title: 'The Hobbit',
+    author: 'J.R.R. Tolkien',
   },
 ];
 
 // Books reducer
-export default function bookReducer(state = initialState, action = {}) {
-  switch (action.type) {
+export default function bookReducer(state = initialState, { type, book }) {
+  switch (type) {
```
Comment (ahmetbozaci) [inline src/redux/books/books.js:23]: - [ ] Well done with adding reducer, action, and types but ı cant add any books on the page. I should add and remove the book on the page. Kindly fix it.

File: src/redux/books/books.js [hunk 2]
```diff
@@ -1,42 +1,45 @@
...
 const REMOVE_BOOK = 'REMOVE_BOOK';
 
 // Define an initial state
-const initialState = [
+export const initialState = [
+  {
+    id: uuidv4(),
+    title: 'The Lord of the Rings',
+    author: 'J.R.R. Tolkien',
+  },
```
Comment (ahmetbozaci) [inline src/redux/books/books.js:13]: - [ ] You added a few books in the initial state but the page shows a different book. Kindly fix it. [Requirement](https://github.com/microverseinc/curriculum-react-redux/blob/main/bookstore/use_redux_in_react_components.md#:~:text=Make%20sure%20that%20list,to%20consume%20the%20state.)

image

**Review by ahmetbozaci — CHANGES_REQUESTED** [review]
# STATUS: Changes Required  ♻️

 @stanleySimeon  ✋🏼 

#### Your project is great. You are almost done. However, there are some issues you should fix. Please fix them then your project will be approved.

___ 

### 💯  Nice to have 💯
- Your git-flow is correct. ✔️
- You set up linter files and no errors. ✔️
- Your pull request is very professional. It has a meaningful title and summary. Keep it up.✔️
___


### ♻️ Request changes ♻️

- [ ] _Check the comments under the review._
- [ ] Kindly fix the broken link in the license section in the readme file.

___

### Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._
___

### Cheers and Happy coding! 💻 👏
almost


___

_Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification._

_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._


___

_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form.](https://airtable.com/shrQAqnBwek5a0O0s)_

[GitHub](https://github.com/ahmetbozaci) | [LinkedIn](https://linkedin.com/in/ahmetbozaci) | [Twitter](https://twitter.com/ahmtbozaci)

_Reviewed by [Ahmet Bozacı ](https://github.com/ahmetbozaci)_

**Review by brytebee — COMMENTED** [review]
Hi, @stanleySimeon👋,

# STATUS: APPROVED 🕺🥳 

Congratulations🍾🥳!! Your project is ready for presentation!  There is nothing else to say other than... it's time to merge it :shipit: 



Good luck! 🎉 

- [ ] **Optional:** Note that branches can only be merged when your code is approved. In future projects, do not merge branches that are still being developed.

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me @brytebee in your question so I can receive the notification.
### _**Use @brytebee on slack for faster responses.**_

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Comment by stanleySimeon** [comment]
Thank you so much

**Pull request merged**